cmd/internal/config/core.go

```diff
@@ -48,6 +50,7 @@ func Init() error {
 
 	// Load the shared config
 	Socket = viper.GetString(SocketKey)
+	Embedded = viper.GetBool(EmbeddedKey)
```

Comment:
I think `config.Embedded` will also be set if it is set in `wash.yaml` right? Is that intentional?

Reply:
It's not a scenario I'm concerned about. And I don't think it will be, this is all evaluated before the config file is loaded.

Reply:
Ok. I asked b/c I was wondering if it would make sense to read directly from the environment variable (and also do the same for the socket). The latter may be worthwhile, b/c then we don't have to say "Don't put your socket overrides in wash.yaml" in our documentation b/c the only way it's read is from the environment.

That doesn't have to be a part of this PR though.